inc/customizer/site-identity.php

```diff
@@ -64,6 +64,30 @@ public function toggle_primer_author_credit() {
 	public function customize_register( WP_Customize_Manager $wp_customize ) {
 
 		$wp_customize->add_setting(
+			'copyright_text',
+			array(
+				'sanitize_callback' => 'wp_kses_post',
```

Comment:
@EvanHerman Should we add `sanitize_js_callback => 'wp_kses_post'` here as well.